### ISSUE TITLE
"top sites" location, one-handed browsing enhancement. 

### ISSUE THREAD

**Issue by Woodcat64** [mozilla-mobile/fenix#14536]
### What is the user problem or growth opportunity you want to see solved?

When toolbar is set to be at the bottom, 'top sites' short cuts remain at the top

### How do you know that this problem exists today? Why is this important?

Installed latest version 79.0.5. Shortcuts are hard to reach with one hand
### Who will benefit from it?

Anyone with bigger phone or small hands

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-6050)

**Comment by madb1lly** [comment]
Hi @Woodcat64,

This is a very good suggestion. I currently find it difficult to reach top sites one-handed with a 5.2" 9:16 screen on a phone with fairly small bezels (Nokia 8). Considering the trend for larger and larger phones means I will find it difficult to get a new phone this small when this one eventually becomes irreparable this means I won't be able to access top sites at all then.

Does that make sense? 

In summary, I think this issue will become more severe with industry trends for smartphone design.

Cheers 🙂

**Comment by boek** [comment]
Going to close this as a dup of https://bugzilla.mozilla.org/show_bug.cgi?id=1813933
